torchaudio/functional/functional.py

```diff
@@ -176,10 +188,20 @@ def inverse_spectrogram(
         Tensor: Dimension `(..., time)`. Least squares estimation of the original signal.
     """
 
+    if type(normalized) is str and normalized not in ["frame_length", "window"]:
```

Comment:
To continue supporting TorchScript, need to perform type refinement.
See example like;

https://github.com/pytorch/text/blob/ca2e5a4601dcaf08bcb7171a3feebb2ecd86d230/torchtext/functional.py#L26-L38